Assets/Mirror/Editor/Weaver/CompilationFinishedHook.cs

```diff
@@ -151,62 +117,24 @@ static void OnCompilationFinished(string assemblyPath, CompilerMessage[] message
             HashSet<string> dependencyPaths = new HashSet<string>();
-
-            // found this assembly in assemblies?
-            if (targetAssembly != null)
-            {
-                // get all dependencies for the target assembly
-                AssemblyName[] dependencies = targetAssembly.GetReferencedAssemblies();
-
-                // does the target assembly depend on Mirror at all?
-                // otherwise there is nothing to weave anyway.
-                bool usesMirror = dependencies.Any(dependency => dependency.Name == MirrorRuntimeAssemblyName);
-                if (!usesMirror)
-                {
-                    return;
-                }
-
-                // get all the directories
-                dependencyPaths = GetDependencyDirectories(dependencies);
-            }
-            else
-            {
-                // Target assembly not found in current domain, trying to load it to check references
-                // will lead to trouble in the build pipeline, so lets assume it should go to weaver.
-                // Add all assemblies in current domain to dependency list since there could be a
-                // dependency lurking there (there might be generated assemblies so ignore file not found exceptions).
-                // (can happen in runtime test framework on editor platform and when doing full library reimport)
-                dependencyPaths = GetNonDynamicAssemblyDirectories(assemblies);
-            }
-
-            // add compiled refs from CompilationPipeline
+            dependencyPaths.Add(Path.GetDirectoryName(assemblyPath));
             foreach (UnityAssembly unityAsm in _cachedAssemblies)
             {
                 if (unityAsm.outputPath != assemblyPath) continue;
 
                 foreach (string unityAsmRef in unityAsm.compiledAssemblyReferences)
                 {
-                    // including NetStandard dependencies causes a stack overflow
-                    // in the Weaver: https://github.com/vis2k/Mirror/issues/791
-                    // root issue: https://github.com/jbevain/cecil/issues/573
-                    if (!unityAsmRef.Contains("NetStandard"))
-                        dependencyPaths.Add(Path.GetDirectoryName(unityAsmRef));
+                    dependencyPaths.Add(Path.GetDirectoryName(unityAsmRef));
                 }
             }
-
-            //if (UnityLogEnabled) Debug.Log("Weaving: " + assemblyPath); // uncomment to easily observe weave targets
-
+             
             // passing null in the outputDirectory param will do an in-place update of the assembly
             if (Program.Process(unityEngineCoreModuleDLL, mirrorRuntimeDll, null, new[] { assemblyPath }, dependencyPaths.ToArray(), HandleWarning, HandleError))
             {
                 WeaveFailed = false;
-                Debug.Log("Weaving succeeded for: " + assemblyPath);
+                //Debug.Log("Weaving succeeded for: " + assemblyPath);
```

Comment:
why comment this?

Reply:
console noise is really annoying, I think. and we are just spamming users. i'd be happy for the average user not to know of weaver's existence. by hook or crook, this thing is not going to be as fragile as it has been. we don't need to constantly tell users "everything is OK", or if we do then man we really messed up in the past.